### DISCUSSION TITLE
SKR 2

### DISCUSSION THREAD

**Discussion by scaramonga** [Jyers/Marlin#1113] [General Discussion]
So, SKR 2 board arrived today, and there will no hope of getting this firmware working on it, as its' a big 'secret' on how to get this firmware working on any of the SKR boards ;)  So I won't ask :)

Seems I got the new revision 'B', well I hope so?, seems to have the new mosfet?

![2016_0101_000638_002](https://user-images.githubusercontent.com/83183034/124516554-d9674e80-ddd9-11eb-9fb4-8077ed3b6404.JPG)

Back to stock Marlin it is then, pity.

**Comment by smooreace** [comment]
there are at least a couple of people that I know were working on the tweaks needed. There is no reason to think its not possible though... its the same firmware!

**Reply by scaramonga** [reply]
It is the same firmware, yes, but lots of changes fall down on the SKR boards, aka, the hardware is not happy with certain changes implemented, and can be a minefield for one compiling for this platform.

**Comment by smooreace** [comment]
It all depends on what you want. Nothing is impossible

**Reply by scaramonga** [reply]
Yeah.

Got it all compiled and working great, albeit not Jyers, but there ya go, it works.  I'm happy :)  I'm sure someone 'nice' will come along and show us how it's all done eventually with this great firmware?, that I have no doubt ;)

**Comment by smooreace** [comment]
The firmware is essentially the same. Everything you did changed in the
stock marlin firmware is the same crap to change in Jyers. In fact, the
config files should compile with Jyers. The only hard part is the pin
changes required to connect the v2 display. And that has been documented
here already

**Reply by scaramonga** [reply]
The config files do not compile with Jyers, always some errors, no matter what one does. (was same with 1.4 Turbo board)  I'm not using the V2 screen either, I'm using a TFT35 E3 v3.

**Comment by scaramonga** [comment]
Yeah, not having much luck with this board either :(

BTT TFT 35 connection issues, and my BLTouch just refuses to work, no matter what wiring I have, or have tried?

Could all this be a 'grounding' problem?  I have the board in a printed case, which lies separate from the main unit, so I'm beginning to think that is the case (pardon the pun).  Dunno?  I'm at my wits end now.  1.4 Turbo was the same?

I have checked, and double checked all wiring, but to no avail.

**Comment by scaramonga** [comment]
Managed to get all problems solved, but BLTouch is a no go ;(  So many different wiring configurations out there for this device, different colours, cables swapped at will, clones with even more heartache, etc.  Have a new one coming later today, so at least I'll be able to know if it's the device itself?, and if not, then I own 2 of them ;), and I can stick one in each ear, then when I'm pissed, I'll be as level as anything.

**Comment by scaramonga** [comment]
To all.

Never get this board, that is all I will say ;)

Ripley signing out.

**Comment by scaramonga** [comment]
Just in case anyone is interested? lol  Got it all figured out :)

Creality probes are configured like this for 32bit boards.  White/Black/Yellow (Servo) - Red/Blue (Probe) (left to right) - for 32bit SKR 2 board. (basically a reverse of the wiring completely), so you have to change to match at other end.  Works fine now :)  Why?, the reason of which escapes me.  FYI.

Your welcome :)

**Comment by smooreace** [comment]
Thats documented all over the internet.

**Comment by scaramonga** [comment]
If it is, then why didn't you say so when I asked? ;), or is it because I should know, and not disturb you without payment?

Did you even come in and tell me so? beforehand.

You are an idiot, and I don't wish to converse with you further.  An arrogant fucker, and if I could get my hands on you, you certainly wouldn't be here ;)  So be thankful, you nasty piece of work.

Never come into any of my posts again, as you add nothing but shit, because that is exactly what you are, and I've forgotten more than you will ever know :)  At 59 years of age I don't need to be talking down to anyone really? ;). as I've been around the block ;)

This is all documented all over the internet.
